app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -37,58 +41,33 @@ EventAdapter createEventAdapter() {
     }
 
     @Override
-    void fetchEvents() {
-        final DateTime now = DateTime.now();
-        DateTime start = now.minusMonths(2).dayOfMonth().withMinimumValue();
```

Comment:
We don't need this anymore right? The API endpoint will always give us the most recent events and as you scroll down you'll get more events.